### PR TITLE
Add suffixes to typos from "exi" to "exu"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -23340,13 +23415,31 @@ exproted->exported
...
+exracted->extracted, exacted,
+exracting->extracting, exacting,
+exraction->extraction
+exractions->extractions
+exractor->extractor
+exractors->extractors
+exracts->extracts, exacts,
 exrension->extension
 exrensions->extensions
+exress->express, excess,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:23437]: Any typos of `egress` out there?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -23340,13 +23415,31 @@ exproted->exported
...
 exressed->expressed
+exresses->expresses
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:23439]: Again `egresses` and more importantly `excesses`?

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:23437]: I couldn't find this typo.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:23439]: Added `excesses`

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**